DESCRIPTION

```diff
@@ -62,7 +62,6 @@ Encoding: UTF-8
 Language: en-US
 LazyData: true
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.2.3
```

Comment:
I think at the moment, we use 7.3.0

Reply:
Why was this changed @jcheng5, @pawelru @cicdguy ? Are new features actually being used? At the very least it should probably be updated again to 7.3.1 which looks to contain a number of bugfixes of various significance. 

Also, fyi, because 7.3.0 is now not the latest CRAN release and so is only available in source form, AND it requires c++11 features, its pretty non-trivial to install (it currently won't install for me, I'll keep trying but its a lot of wheel spinning thats not really useful to anyone).

Reply:
Yep, sorry about that other Joe. my mistake there.

I think we should do that update, it seems 7.3.1 is purely bugfixes on top of 7.3.0. 

Also, @cicdguy is there anywhere convenient that the manifest for the images is published?  If not, its probably a good idea since roxygen mismatches cause check failures  the iteration overhead both for the contributor and the nest team is non-trivial. I'm not exactly sure where the best place to put that would be, though, as it would apply across all NEST projects, right? (though I don't know how many contributed PRs the other projects are getting...)